src/views/Clients/ViewClients/index.jsx

```diff
@@ -39,6 +39,10 @@ export default class ViewClients extends PureComponent {
     clientSearch: '',
     // eslint-disable-next-line react/no-unused-state
     previousClientId: '',
+    // eslint-disable-next-line react/no-unused-state
+    closeError: true,
+    // eslint-disable-next-line react/no-unused-state
+    error: false,
```

Comment:
Instead of having to implement this kind of logic in every component, what do you think about creating a new component `components/ErrorPanel` that would take care closing errors and re-displaying the error if a new one is given. Something like the following (not tested):

```jsx
import React, { Component } from 'react';
import { oneOfType, string, object } from 'prop-types';
import MuiErrorPanel from '@mozilla-frontend-infra/components/ErrorPanel';

export default class ErrorPanel extends Component {
  static propTypes = {
    /** Error to display. */
    error: oneOfType([string, object]),
  };

  static defaultProps = {
    error: null,
  };

  state = {
    error: null,
    // eslint-disable-next-line react/no-unused-state
    previousError: null,
  };

  static getDerivedStateFromProps(props, state) {
    if (props.error !== state.previousError) {
      return {
        error: props.error,
        previousError: props.error,
      };
    }

    return null;
  }

  handleErrorClose = () => {
    const { onClose } = this.props;

    this.setState({ error: null });

    if (onClose) {
      onClose();
    }
  };

  render() {
    const { error: _, onClose, ...props } = this.props;
    const { error } = this.state;

    return (
      error && (
        <MuiErrorPanel
          error={error}
          onClose={this.handleErrorClose}
          {...props}
        />
      )
    );
  }
}
```